purchase_request_to_rfq/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request to RFQ",
+    "author": "Eficent Business and IT Consulting Services S.L., "
```

Comment:
We did help a bit but we certainly don't claim any authorship. A mention in contributors section is enough.